rumqttc/src/eventloop.rs

```diff
@@ -81,7 +85,7 @@ impl EventLoop {
     ///
     /// When connection encounters critical errors (like auth failure), user has a choice to
     /// access and update `options`, `state` and `requests`.
-    pub fn new(options: MqttOptions, cap: usize) -> EventLoop {
+    pub fn new(options: MqttOptions, network_options: NetworkOptions, cap: usize) -> EventLoop {
```

Comment:
rename `options` to `mqtt_options`

Reply:
i suppose you meant in eventloop as well, right?